mmv1/third_party/terraform/website/docs/r/composer_user_workloads_secret.html.markdown

```diff
@@ -0,0 +1,99 @@
+---
+subcategory: "Cloud Composer"
+description: |-
+  Kubernetes secret in Composer Environment workloads.
```

Comment:
Do we need to mention that this is only for Composer 3 environments?

I do not know what approach we follow, but that maybe not clear for the user that this resource will only work for some specific versions of Composer.